Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp

```diff
@@ -217,19 +216,22 @@ static dd4hep::Ref_t createHCal(dd4hep::Detector& lcdd, xml_h xmlElement, dd4hep
 
   for (unsigned int idxZRow = 0; idxZRow < numSequencesZ; ++idxZRow) {
     double zOffset = -dzDetector + dZEndPlate + space + (2 * idxZRow + 1) * (dzSequence * 0.5);
-    lLog << MSG::DEBUG << "z offset of wedges = " << zOffset << std::endl;
+    // unsigned int seqId = idxZRow % 3;
+    lLog << MSG::DEBUG << "z offset of wedges = " << zOffset << endmsg;
 
     if ((-dzDetector + zOffset) >= dzDetector) {
-      lLog << MSG::WARNING << " WARNING!!!! Module position outside of detector envelope" << std::endl;
+      lLog << MSG::WARNING << " WARNING!!!! Module position outside of detector envelope" << endmsg;
     }
     dd4hep::Position wedgeOffset(0, zOffset, 0);
+    // dd4hep::Transform3D rotate(dd4hep::RotationY(seqId*-0.5 * dd4hep::pi), wedgeOffset);
     // Fill vector for DetElements
     rows.push_back(moduleVolume.placeVolume(wedgeVolume, wedgeOffset));
     rows.back().addPhysVolID("row", idxZRow);
   }
 
   for (unsigned int idxPhi = 0; idxPhi < numSequencesPhi; ++idxPhi) {
-    double phi = 0.5 * dphi + idxPhi * dphi;  // 0.5*dphi for middle of module
+    double phi = -0.5 * dd4hep::pi - dphi * 0.5 -
+        idxPhi * dphi;  // modules placed following phi-eta segmentation with offset: -pi
```

Comment:
Remove the comment from the back or adjust it so it doesn't indicate you use eta-phi segmentation.